package.json

```diff
@@ -14,7 +14,8 @@
     "unlink": "node ./scripts/unlink.js",
     "build:dev": "npm i && npm run clean && npm run link && npm run build -- -ni && npm run build:demo",
     "build:library": "node ./scripts/build.js --library",
-    "build:documentation": "node ./scripts/build.js --documentation",
+    "build:documentation": "npm run build:dev && npm run transfer:documentation",
+    "transfer:documentation": "node scripts/transfer-doc-files.js",
```

Comment:
`--component` argument does not get passed to the `transfer:documentation`. It always builds all components.
Previously the command with the options was `npm run build:documentation -- --component checkbox`.

Reply:
Also we should update the README.md

Reply:
* Removed trasfer:documentation from the package.json scripts. Now everything is controlled via build:documentation.
* Now the next commands are working.
    * --rebuild will trigger `npm run build:dev`
    * --component will transfer just the files for the specified component
    * If none of this flags are set then all the components will be transferred to the documentation
    * If just the --rebuild exists it will rebuild all components via `npm run build:dev` and then transfer all the components files to the documentation. 
```
npm run build:documentation -- --rebuild
npm run build:documentation -- --component accordion-menu
```
* Update README.md
* Updated the documentation files

Reply:
Works exactly how you described it. Just mentioning that it doesn't work with PowerShell unfortunately.